components/hab/src/command/pkg/binlink.rs

```diff
@@ -301,7 +300,7 @@ mod test {
               rootfs.path(),
               force).unwrap();
         #[cfg(windows)]
-        assert!(fs::read_to_string(rootfs_bin_dir.join(magicate_link)).unwrap().contains(&format!("PATH={}{}", rootfs_src_dir.to_string_lossy(), curr_path)));
+        assert!(fs::read_to_string(rootfs_bin_dir.join(magicate_link)).unwrap().contains(&format!("PATH={}{}", rootfs_src_dir.to_string_lossy(), ";%PATH%")));
```

Comment:
I wonder if we need to start running `rustfmt` in CI on Windows, too... Is this line allowed to stay this long because it's only on Windows? :thinking:

Reply:
when I run rustfmt locally on windows ^^^ is the result

Reply:
`rustfmt` doesn't interpret `#[cfg(windows)]`. When I run it locally, it still reformats things in those blocks. I'm pretty sure it operates at the AST level and ignores conditional compilation other than `rustfmt`-specific directives.